polling.go

```diff
@@ -0,0 +1,76 @@
+package ldclient
+
+import (
+	"sync"
+	"time"
+)
+
+type pollingProcessor struct {
+	store              FeatureStore
+	requestor          *requestor
+	config             Config
+	setInitializedOnce sync.Once
+	isInitialized      bool
+	lastHeaders        *cacheHeaders
+	quit               chan bool
+}
+
+func newPollingProcessor(config Config, store FeatureStore, requestor *requestor) updateProcessor {
+	pp := &pollingProcessor{
+		store:     store,
+		requestor: requestor,
+		config:    config,
+		quit:      make(chan bool),
+	}
+
+	return pp
+}
+
+func (pp *pollingProcessor) start(ch chan<- bool) {
+	go func() {
+		for {
+			select {
+			case <-pp.quit:
+				return
+			default:
+				then := time.Now()
+				err := pp.poll()
+				if err == nil {
+					pp.setInitializedOnce.Do(func() {
+						pp.isInitialized = true
+						ch <- true
+					})
+				}
+				delta := (1 * time.Second) - time.Since(then)
```

Comment:
Do we want to dictate this polling interval or does it make sense to let our customers set it in the config? We definitely don't want this number too small, so we'd have to make sure it isn't abused.